internal/pkg/rpaas/nginx/configuration_render.go

```diff
@@ -127,227 +161,182 @@ var rawNginxConfiguration = `
 # This file was generated by RPaaS (https://github.com/tsuru/rpaas-operator.git)
 # Do not modify this file, any change will be lost.
 
-user {{with .Config.User}}{{.}}{{else}}nginx{{end}};
-worker_processes {{with .Config.WorkerProcesses}}{{.}}{{else}}1{{end}};
+{{- with $config.User }}
+user {{ . }};
```

Comment:
With this change the default user is now `nobody` instead of nginx. Is there an impact of this change with the default nginx image we recommend?

Reply:
This directive is really important when starting the nginx server as root user. As this configuration template is based on `tsuru/nginx-rpaas` container image, we don't need to state it since the nginx server is started by nginx user by default.